website/docs/r/scc_scope.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_scope"
+description: |-
+  Manages scc_scope.
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_scope
+
+Create, update, and delete scc_scopes with this resource.
+
+## Example Usage
+
+```hcl
+resource "ibm_scc_scope" "scc_account_scope" {
+  description = "This scope allows a profile attachment to target an IBM account"
+  environment = "ibm-cloud"
+  instance_id = "b36c26e9-477a-43a1-9c50-19aff8e5d760"
+  name        = "Sample account Scope"
+  properties {
+    account_id = "8e042beeccee40748674442960b9eb34"
+  }
+}
+
+resource "ibm_scc_scope" "scc_enterprise_scope" {
+  description = "This scope allows a profile attachment to target an IBM enterprise"
+  environment = "ibm-cloud"
+  instance_id = "b36c26e9-477a-43a1-9c50-19aff8e5d760"
+  name        = "Sample account Scope"
+  properties {
+    enterprise_id = "6a204bd89f3c8348afd5c77c717a097a"
+  }
+}
+
+resource "ibm_scc_scope" "scc_ibm_facts_scope" {
+  description = "This scope allows a profile attachment to target a facts provider instance"
+  environment = "ibm-cloud"
+  instance_id = "b36c26e9-477a-43a1-9c50-19aff8e5d760"
+  name        = "Sample facts Scope"
+  properties {
+    ibm_facts_api_instance_id = ibm_scc_provider_type_instance.facts_provider_type_instance.provider_type_instance_id
+  }
+}
+```
+
+## Argument Reference
+
+You can specify the following arguments for this resource.
+
+* `description` - (Optional, String) The scope description.
+  * Constraints: The maximum length is `512` characters. The minimum length is `0` characters. The value must match regular expression `/^[a-zA-Z0-9_,'\\s\\-\\.]*$/`.
+* `environment` - (Optional, String) The scope environment. This value details what cloud provider the scope targets.
+  * Constraints: The maximum length is `128` characters. The minimum length is `0` characters. The value must match regular expression `/^[a-zA-Z0-9_,'\\s\\-\\.]*$/`.
+* `instance_id` - (Required, Forces new resource, String) The ID of the Security and Compliance Center instance.
+  * Constraints: The maximum length is `36` characters. The minimum length is `36` characters. The value must match regular expression `/^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}$/`.
+* `name` - (Optional, String) The scope name.
+  * Constraints: The maximum length is `128` characters. The minimum length is `1` character. The value must match regular expression `/^[a-zA-Z0-9_,'\\s\\-\\.]*$/`.
+* `properties` - (Optional, List) The span for the scope to target.
+Nested schema for **properties**:
+    * `account_id` - (Optional, ForceNew, String) The ID of the IBM account ID.
```

Comment:
schema is not right

Reply:
The schema of a Terraform resource does not need to be the same as the schema in an openapi doc. If the Terraform code were to have the same exact schema as the openapi spec, the Terraform code will never work. The values are polymorphic and do not have strongly define types in the attributes. Having a schema that is different does not necessitate a simple "schema is not right" comment and should be explained more throughly.